pom.xml

```diff
@@ -25,6 +25,7 @@
   <properties>
     <apacheds.version>2.0.0-M23</apacheds.version>
     <apacheds.jdbm.version>2.0.0-M3</apacheds.jdbm.version>
+    <jenkins.version>1.648</jenkins.version>
```

Comment:
Why this version? Generally we prefer to use an LTS as the baseline; the next would be 1.651.x. (My preference is to depend on 1.651.3, the last in the series; some others advocate for the weekly 1.651. Whatever.)

BTW `Jenkinsfile` currently requests testing against the baseline, and 2.32.3. May as well make that 2.73.1 so we can catch possible PCT issues earlier.

Reply:
(But note that only people with write permission on the repo can test the effect of changes to `Jenkinsfile`, so perhaps that needs to wait for a follow-up.)